_posts/2018-01-22-threads.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+author: Santiago Doldán
+title: Introduce threads to Suttna
```

Comment:
Introducing threads to Suttna

Reply:
I think `Introducing Threads` is better :)